modules/ocf_mirrors/manifests/ftpsync.pp

```diff
@@ -1,20 +1,16 @@
 # ftpsync is a Debian tool for reliably mirroring Debian archives:
 # https://www.debian.org/mirror/ftpmirror
 define ocf_mirrors::ftpsync(
-    $cron_minute,
     $rsync_host,
+    $cron_minute = undef,
```

Comment:
is there ever a case we won't want a cron minute? i figure even with push mirroring, we'd still want to sync at least once a day, just in case